mypy/checker.py

```diff
@@ -796,6 +796,7 @@ def check_reverse_op_method(self, defn: FuncItem, typ: CallableType,
                 arg_types = list(arg_type.iter_deep())
             else:
                 return
+            # We check that each method is fine when dispatched on a proper self argument
```

Comment:
Neither. I meant that I would prefer to see a single commit pushed to the PR that included the previous commit plus this one (which just added a comment that you forgot on the previous commit).

You can do that locally by using `git commit --amend` for the second (which just adds a comment you forgot in the previous local commit). You can also do this using `git rebase -i`.